src/edit_distance.jl

```diff
@@ -78,17 +81,83 @@ function edit_distance(G₁::AbstractGraph, G₂::AbstractGraph;
             vs = setdiff(1:nv(G₂), [v for (u, v) in λ])
 
             if k < nv(G₁) # there are still vertices to process in G₁?
+                # u be the next node to be transformed
+                u = k+1
+                # transformation operations on neighbor of u 
+                u_edited_neighbors = Vector{Tuple{Int,Int}}()
+                u_neighbors = all_neighbors(G₁, u)
+                
+                # # add transformation (x, y) to u_edited_neighbors if x is neighbor of u
```

Comment:
So `u_dited_neighbors` holds neighbors or transformations? It is a little confusing the variable content does not match its name.

Reply:
It holds the transformation, yes the name is indeed confusing